inc/saiport.h

```diff
@@ -74,6 +74,23 @@ typedef enum _sai_port_oper_status_t
 
 } sai_port_oper_status_t;
 
+/**
+ * @brief Attribute bitmap data for #SAI_PORT_ATTR_ERROR_STATUS
+ */
+typedef enum _sai_port_error_status_t
+{
+    /** No errors */
+    SAI_PORT_ERROR_STATUS_CLEAR = 0,
+
+    SAI_PORT_ERROR_STATUS_MAC_LOCAL_FAULT = 1,
+
+    SAI_PORT_ERROR_STATUS_MAC_REMOTE_FAULT = 2,
+
+    SAI_PORT_ERROR_STATUS_FEC_SYNC_LOSS = 4,
+
+    SAI_PORT_ERROR_STATUS_FEC_LOSS_ALIGNMENT_MARKER = 8
+} sai_port_error_status_t;
```

Comment:
please add SAI_PORT_ERROR_STATUS_HIGH_SER and SAI_PORT_ERROR_STATUS_HIGH_BER

Reply:
@eddyk-nvidia added